_data/events.yml

```diff
@@ -1516,11 +1516,23 @@
   githubUrl: "https://github.com/itenium-be/Chatbot-Bootcamp"
 
 
-# TODO: events to add? (there are also FunEvents in between here!)
-# 2022-06-29 co2duino
-
-
-# Socials
-# Battle Cart: https://www.instagram.com/p/CamC_sOrgJr & https://x.com/itenium_be/status/1498935932938031106?s=20
-# Serverless: https://x.com/itenium_be/status/1497130816580771847?s=20
-# Flutter: https://x.com/itenium_be/status/1433769104574459904?s=20
+- name: "Serverless"
+  date: 2022-02-25
+  track: "FrontendTrack"
```

Comment:
TechEvent

Reply:
Hmm or ArchitectureTrack
Typically this is in the pptx (but maybe back then this was not yet the case)